server/api/src/rs_get_resource_info_for_operation.cpp

```diff
@@ -0,0 +1,71 @@
+#include "irods/get_resource_info_for_operation.h"
+
+#include "irods/getRemoteZoneResc.h"
+#include "irods/irods_logger.hpp"
+#include "irods/irods_resource_backport.hpp"
+#include "irods/irods_resource_redirect.hpp"
+#include "irods/rodsError.h"
+
+#include <nlohmann/json.hpp>
+#include <fmt/format.h>
+
+#include <array>
+#include <cstring> // for strdup
+#include <string>
+
+int rs_get_resource_info_for_operation(rsComm_t* _rsComm, dataObjInp_t* _dataObjInp, char** _out_info)
+{
+    using log_api = irods::experimental::log::api;
+    if (_rsComm == nullptr || _dataObjInp == nullptr || _out_info == nullptr) {
+        return SYS_INVALID_INPUT_PARAM;
+    }
+    rodsServerHost_t* rodsServerHost = nullptr;
+    const int remoteFlag = getAndConnRemoteZone(_rsComm, _dataObjInp, &rodsServerHost, REMOTE_OPEN);
+    if (remoteFlag < 0) {
+        return remoteFlag;
```

Comment:
Consider logging a message.

Reply:
Logging now, pleasee lmk if this is ready to resolve.

Reply:
Looks good.